.github/workflows/client-tests.yml

```diff
@@ -42,6 +36,31 @@ jobs:
         go-version: 1.14
```

Comment:
Could we run 1.17 here?

Reply:
This is go version per branch. So 2.9 is 1.14, develop is 1.17, once we move this to develop we can then swap it over :)

Reply:
Perfect.